docs/source/conf.py

```diff
@@ -62,3 +62,28 @@
 
 # Source files supported by Sphinx.
 source_suffix = [".rst", ".md"]
+
+autosummary_generate = True
+
+# A way to automatically generate API documentation upon push to GitHub.
+# https://github.com/readthedocs/readthedocs.org/issues/1139
+def run_apidoc(_):
+    ignore_paths = []
+
+    argv = ["-f", "-T", "-e", "-M", "-o", "source/generated", ".."] + ignore_paths
```

Comment:
`"source/generated"` is the output directory if you want to build it locally, just like it was previously